site/layouts/shortcodes/example.html

```diff
@@ -19,36 +19,10 @@
 {{- end -}}
 
 {{- if eq $show_markup true -}}
-  {{- $.Scratch.Set "highlight_content" $input -}}
-
-  {{- if (strings.Contains $input `<svg class="bd-placeholder-img`) -}}
-    {{- $.Scratch.Set "highlight_content" "" -}}
-
-    {{- $modified_content := replace $input `<svg class="bd-placeholder-img` `✂️<svg class="bd-placeholder-img` -}}
-    {{- $modified_content = replace $modified_content "</svg>\n" "</svg>✂️" -}}
-    {{- $modified_content = split $modified_content "✂️" -}}
-
-    {{- range $content_chunk := $modified_content -}}
-      {{- $image_class := "" -}}
-
-      {{- if (strings.Contains $content_chunk `<svg class="bd-placeholder-img `) -}}
-        {{- $image_class = replace $content_chunk "bd-placeholder-img " "bd-placeholder-img ✂️ " -}}
-        {{- $image_class = split (replace $image_class `" width="` `✂️" width="`) "✂️" -}}
-        {{- $image_class = replace (index $image_class 1) "bd-placeholder-img-lg" "" -}}
-        {{- $image_class = trim $image_class " " -}}
-      {{- end -}}
-
-      {{- if (strings.Contains $content_chunk `<svg class="bd-placeholder-img`) -}}
-        {{- if $image_class -}}
-          {{- $.Scratch.Add "highlight_content" (printf `<img src="..." class="%s" alt="...">` $image_class) -}}
-        {{- else -}}
-          {{- $.Scratch.Add "highlight_content" `<img src="..." alt="...">` -}}
-        {{- end -}}
-      {{- else -}}
-        {{- $.Scratch.Add "highlight_content" $content_chunk -}}
-      {{- end -}}
-    {{- end -}}
-  {{- end -}}
+  
+  {{- $content := replaceRE `(class="\s*?")` `` (replaceRE `<svg.*class=.bd\-placeholder\-img(?:\-lg)?(?:\s*?bd\-placeholder\-img\-lg)?\s?(.*?)".*?<\/svg>\n` `<img src="..." class="$1" alt="..." >` $input) -}}
+    
+  {{- $.Scratch.Set "highlight_content" $content -}}
```

Comment:
A simple variable assignment should work now, instead of Scratch. Can you try it?